### DISCUSSION TITLE
bw-marker2.py does not work for all PDF's

### DISCUSSION THREAD

**Discussion by dgrunspan** [pymupdf/PyMuPDF#1443] [Looking for help]
bw-marker2.py does not work for all PDF's

Try this big one and see
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5766240/pdf/pone.0191194.pdf

I don't need 100% guarantee , but at least a sign it did not work and then I can leave this file alone and don't compress it

**Comment by JorjMcKie** [comment]
This script was developed to help someone in a specific situation.
It is not part of PyMuPDF's official code, so you are on your own to develop it into something useful in your situation.
When done, please make it available via a PR to the repository / folder https://github.com/pymupdf/PyMuPDF-Utilities/tree/master/examples.

**Comment by JorjMcKie** [comment]
I may have been a little harsh with my response, but please consider that I am on the other end of a mobile phone internet connection - thus paying per every byte of internet traffic.
So downloading an 85 MB animal just to confirm that some script (developed on a sidetrack) doesn't work for it, is too much to ask - sorry.

**Comment by dgrunspan** [comment]
No problem. Thx







בתאריך יום ה׳, 9 בדצמ׳ 2021, 14:31, מאת Jorj X. McKie ‏<
***@***.***>:
